src/components/Transaction/TransactionsDataTable.vue

```diff
@@ -840,8 +840,8 @@ export default defineComponent({
 
 }
 
-::v-deep .v-data-table > .v-data-table__wrapper>table {
-  min-height: 215px;
+::v-deep .transaction-list .v-data-table__wrapper {
+  min-height: 240px;
```

Comment:
Patty found issue in firefox where more actions are cut off. This resolves the issue for both chrome and firefox. Previous CSS didn't affect firefox.